--- apache.conf.orig	2012-11-24 16:03:19.000000000 +0100
+++ apache.conf	2017-03-05 17:16:55.679278263 +0100
@@ -9,9 +9,9 @@
    	#
    	# Use the same value as defined in nagios.conf
    	#
-   	AuthName "Nagios Access"
+   	AuthName "Icinga Access"
    	AuthType Basic
-   	AuthUserFile /etc/nagios3/htpasswd.users
+   	AuthUserFile /etc/icinga/htpasswd.users
    	Require valid-user
 	<IfModule mod_rewrite.c>
 		# Turn on URL rewriting
