--- index.php.orig	2017-03-07 20:19:11.360337880 +0100
+++ index.php	2017-03-07 20:19:42.039321612 +0100
@@ -87,7 +87,7 @@
  * Set the error reporting level. Unless you have a special need, E_ALL is a
  * good level for error reporting.
  */
-error_reporting(E_ALL & ~E_STRICT);
+error_reporting(E_ALL & ~E_STRICT & ~E_DEPRECATED);
 
 /**
  * Turning off display_errors will effectively disable Kohana error display
